package/README.md

```
@@ -85,19 +80,19 @@ interface AudioPlayerProps {
---- | --- | --- 
-`playList` | [PlayList](#playlist) | [ ] 
-`audioInitialState` | [InitialStates](#InitialStates) | isPlaying: false </br>repeatType: "ALL" </br>volume: 1 
-`activeUI` | [ActiveUI](#activeui) | playButton : true
-`customIcons` | [CustomIcons](#customicons) | undefined 
-`coverImgsCss` | [CoverImgsCss](#coverimgscss) | undefined 
-`placement` | [Placement](#placement) | playListPlacement : "bottom" </br>interfacePlacement :[DefaultInterfacePlacement](#default-interface-placement)
-`rootContainerProps` | [RootContainerProps](#rootcontainerprops) | theme: spectrum-theme-default<br/>width: 100% <br/>position: 'static'<br/>UNSAFE_className: rm-audio-player-provider
+| Prop                 | Type                                      | Default                                                                                                         |
+| -------------------- | ----------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| `playList`           | [PlayList](#playlist)                     | [ ]                                                                                                             |
+| `audioInitialState`  | [InitialStates](#InitialStates)           | isPlaying: false </br>repeatType: "ALL" </br>volume: 1                                                          |
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid
Expected: `#initialstates`; Actual: `#InitialStates`

(MD051, link-fragments)